dist/pathfora.js

```diff
@@ -3031,19 +3046,28 @@
    * @params {object} pf
    */
   function validateAccountId (pf) {
+    var acctid;
+
     if (typeof pf.acctid === 'undefined' || pf.acctid === '') {
       if (window.lio && window.lio.account) {
-        if (
-          typeof window.lio.account.id === 'undefined' ||
-          window.lio.account.id === ''
-        ) {
-          throw new Error('Lytics Javascript tag returned an empty account id.');
-        }
-
-        pf.acctid = window.lio.account.id;
+        acctid = window.lio.account.id;
+      } else if (
```

Comment:
nit: can we get the comments here as well regarding legacy vs v3 tag?

Reply:
added to src (this is the dist)